### PR TITLE
Add page about working groups

### PR DESCRIPTION
**Pull request by donatieng**
Add working groups mention in contributing section

### CODE REVIEW

File: docs/contributing/guidelines/working_groups.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Working Groups
+
+Mbed OS has a number of established working groups aimed at improving and building new services and features. The working groups bring together like-minded individuals and teams of similar interests and provide a platform to share feedback and discuss what can be delivered as a community.
```
Comment (adbridge) [inline docs/contributing/guidelines/working_groups.md]: Too many 'and's in that last sentence . I would rephrase as follows:
The working groups bring together like-minded individuals and teams of similar interests, providing a platform to share feedback and discuss what can be delivered as a community.

**Review by adbridge — CHANGES_REQUESTED** [review]

File: docs/contributing/guidelines/working_groups.md (hunk 1 above)
Comment (donatieng) [reply docs/contributing/guidelines/working_groups.md]: Good point, thanks for the suggestion!

**Review by adbridge — APPROVED** [review]
